src/storage/MMKVStorage.ts

```diff
@@ -4,10 +4,10 @@ import { initializeMMKVFlipper } from 'react-native-mmkv-flipper-plugin'
 export type AcceptedValue = boolean | string | number | object
 
 export class MMKVStorage {
-  private id = 'mmkv.default'
-  private storage: MMKV | null = null
+  private id: string
+  private storage: MMKV
 
-  constructor(id = 'mmkv.default', encryptionKey?: string) {
+  constructor(id = 'main_storage', encryptionKey?: string) {
```

Comment:
When I load this PR, the app it got reset and it asks me for my mnemonic, is this because we renamed this database?